src/Twig/RichEditorExtension.php

```diff
@@ -99,7 +115,7 @@ public function renderRichEditorField(string $content)
      *
      * @return string
      */
-    private function renderElements(array $elements): string
+    public function renderElements(array $elements): string
```

Comment:
Why it becomes public ?

Reply:
To be able to use them from filter/function